plugins/gcpaudit/walkthrough.md

```diff
@@ -0,0 +1,389 @@
+
+### GCP Audit Logs walkthrough
+
+In this section, you will learn how to install and use the GCP Audit Logs plugin. You’ll first configure the necessary GCP services. This includes setting up a Pub/Sub topic and subscription and creating a GCP Logging service sink. Then, you will build the plugin. Next, you’ll install Falco and configure it to use the plugin. Finally, you’ll test everything to ensure it’s all working correctly. 
+
+#### Prerequisites
+
+To follow along with this tutorial, you will need a GCP project and an account with appropriate permissions, including the ability to:
+
+
+
+* Create and manage compute instances
+* Create and manage pub/sub topics
+* Access GCP audit logs and create custom GCP Logging sinks
+
+You will also need to have the gcloud command line tool installed on your local machine or use Cloud Shell (which already has gcloud installed). 
+
+Make sure you know the project ID of your GCP project. 
+
+Finally, to build the plugin, you will need to have both Git and Go 1.17 or greater. If you use Cloud Shell, both tools are already installed for you. 
+
+
+#### Step 1: Configure the GCP services
+
+In this section, you’ll configure the necessary backend GCP services. 
+
+**NOTE**: This walkthrough creates resources that could incur charges against your GCP bill. The easiest way to reduce charges is to use a project that can be deleted when you are finished. Alternatively, you will need to delete the resources you created manually upon creation. There are instructions on how to do this at the end of the walkthrough. 
+
+To begin make sure you’re authenticated to your GCP account in gcloud and you’ve selected the appropriate project.
+
+
+```
+    gcloud auth login 
+    gcloud config set project [project id]
+```
+
+
+It is helpful later on to have your GCP project ID configured in an environment variable.
+
+	`PROJECT_ID="[project id]"`
+
+The next step is to create a Pub/Sub topic to receive the output from the GCP Audit logs. 
+
+	`gcloud pubsub topics create falco-plugin-topic`
+
+The plugin uses a Pub/Sub subscription to receive events, so create that now.
+
+	`gcloud pubsub subscriptions create falco-plugin-sub --topic=falco-plugin-topic`
+
+Next, create a GCP Logging sink to filter events and send them over to the Pub/Sub topic you just created. In this example, you use Falco’s recommended sink query, but you can adjust the query as you see fit based on your organization’s needs. 
+
+**NOTE**: You will need to manually add your project ID in the first line of the `--log-filter` parameter (all the quotations make variable interpolation difficult).
+
+
+```
+gcloud logging sinks create falco-plugin-sink pubsub.googleapis.com/projects/$PROJECT_ID/topics/falco-plugin-topic --log-filter='"projects/[PROJECT ID]/logs/cloudaudit.googleapis.com%2Factivity" AND
```

Comment:
```suggestion
gcloud logging sinks create falco-plugin-sink pubsub.googleapis.com/projects/${PROJECT_ID}/topics/falco-plugin-topic --log-filter='"projects/${PROJECT_ID}/logs/cloudaudit.googleapis.com%2Factivity" AND
```

Reply:
@Issif I didn't use the interpolation here because it's already in quotes, which puts ${PROJECT_ID} as a string literal.